### PR TITLE
Update changelog for release

### PR DESCRIPTION
**Pull request by mihaimaruseac**
### Checklist

- [ ] Add a changelog if necessary. See https://keepachangelog.com/ for how to write it.
- [ ] Add tests in [TESTS.md](/TESTS.md) if possible.

### CODE REVIEW

**Comment by toku-sa-n** [comment]
I think the next version should be 6.1.0 as there are breaking changes (like `reformat`'s signature change). After that version, we should be able to increment the patch number as long as we don't modify exposed contents in the `HIndent` module.

**Comment by mihaimaruseac** [comment]
Right, thanks for noticing. Updated

**Review by toku-sa-n — CHANGES_REQUESTED** [review]

File: CHANGELOG.md
[the remark this answers is not in the document]
Comment (mihaimaruseac) [reply CHANGELOG.md:1]: Thanks. Too early for me, missed it

**Comment by mihaimaruseac** [comment]
Updated `package.yaml`, the `.cabal` file was updated before.

**Review by toku-sa-n — APPROVED** [review]
Thank you!

**Comment by toku-sa-n** [comment]
Thanks for the release! Did you update the Hackage one?

**Comment by mihaimaruseac** [comment]
No, I had to leave for work. Just got home and will do the upload now.

**Comment by mihaimaruseac** [comment]
Now it should be at https://hackage.haskell.org/package/hindent-6.1.0 (documentation I think will be updated shortly, it's weird we only have the top level module only)

**Comment by toku-sa-n** [comment]
Thank you!

> it's weird we only have the top level module only

It's expected as we did in #729.

**Comment by mihaimaruseac** [comment]
Oh, my bad, forgot about it.

Nice.

**Pull request merged**